build/build.sh

```diff
@@ -43,3 +44,20 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+echo "Exporting Word Docx manuscript"
+pandoc --verbose \
+--smart \
```

Comment:
I think `--smart` is most likely a good thing for us... but let's try to keep in sync with HTML output as much as possible. So also add to HTML command or save for a future PR.